kdf.go

```diff
@@ -15,8 +15,8 @@ import (
  */
 
 /*
-KDFRFC5054 is *not* recommended. Instead use a KDF that
-involes a hashing scheme designed for password hashing.
+KDFRFC5054 is *not* recommended. Instead use a key derivation function (KDF) that
```

Comment:
Is “KDFRFC5054” a thing? Is there a better term for it? “The KDF in RFC 5054”? Google shows the only usage of “[KDFRFC5054](https://www.google.com/search?q=KDFRFC5054)” is in these srp docs of ours. :)

Reply:
Also, is it better to define KDF before referring to the KDF in RFC 5054?